content/articles/creating-file-management-application-using-winforms/index.md

```diff
@@ -0,0 +1,241 @@
+### Introduction
+File management encompasses everything from generating and editing files to deleting them from a computer's hard drive.
+
+It is crucial to consider when it comes to data that needs to be handled with care and recorded for other purposes such as security and future references.
+
+We shall demonstrate how you can create files and folders, write data to them in your interested formats, read the memory data, delete the initially recorded files, and view all data using WinForms.
+
+### Prerequisites
+1. Basic c# programming language and .NET framework.
+2. A machine with [Visual Studio](https://visualstudio.microsoft.com/vs/community/) running.
+
+### Table of contents
+- [Prerequisites](#prerequisites).
+- [Visual Studio setup](#step-1-visual-studio-setup).
+- [Designing the WinForm](#step-2-designing-the-winform).
+- [Including the necessary namespace](#including-the-necessary-namespace).
+- [Writting the C# code](#step-3-writting-c#-code)
+- [Debugging the application](#step-4-debugging-the-application).
+- [Conclusion](#conclusion).
+
+Let's get started!!
+
+### Step 1. Visual studio setup
+- Open visual studio on your device.
+- Select *`create new project`* in the Get Started window.
+
+![Creating a new project](engineering-education/creating-file-management-application-using-winforms/create-a-new-project.jpg)
```

Comment:
Do this "/engineering-education/creating-file-management-application-using-winforms/" instead of "engineering-education/creating-file-management-application-using-winforms/"

Correct this for all instances...